packages/core/src/validators.ts

```diff
@@ -158,6 +158,12 @@ export class PatternViolation extends Violation {
   }
 }
 
+export class AnyOfMismatch extends Violation {
```

Comment:
AnyOfMismatch is not actually used anywhere, is it?

Reply:
Yeah, that is true. Will remove it at once